requirements.txt

```diff
@@ -1,4 +1,4 @@
-chalice
+chalice==1.2.2
```

Comment:
It is commonly desirable to allow version updates so long at the are unlikely to include breaking changes. If you don't need to lock Chalice specifically to 1.2.2, then I would suggest:
`chalice >= 1.2.2, < 2

Reply:
I had a problem with 1.3 and so pinned to 1.2.2 for now.

Reply:
https://github.com/DataBiosphere/dos-azul-lambda/issues/20

Reply:
Thanks for the info. Might be nice to include that as a comment in the requirements.txt